packages/shared/src/components/notifications/NotificationItem.tsx

```diff
@@ -49,17 +50,32 @@ function NotificationItem({
     return null;
   }
 
+  const filteredAvatars = avatars?.filter((avatar) => avatar) || [];
+
   const avatarComponents =
-    avatars
-      ?.map?.((avatar) => (
-        <NotificationItemAvatar
-          key={avatar.referenceId}
-          className="z-1"
-          {...avatar}
-        />
-      ))
-      .filter((avatar) => avatar) ?? [];
-  const hasAvatar = avatarComponents.length > 0;
+    type === NotificationType.CollectionUpdated ? (
```

Comment:
Is there anyway we can uniform the two to use the ProfilePictureGroup?
(Think only difference is basically the rounded state.

Reply:
Hmm I see the spacing is also overlapping for collecitons which is not for the others.

Reply:
good point, this is already modified on the collection post page branch though: https://github.com/dailydotdev/apps/pull/2370/files#diff-1dbb3736e496fd2b9830576074141b705cc949e7eef12d17eb6c0cb5c77211e9

I'll consolidate with those changes here as well